src/interfaces/action-interface.ts

```diff
@@ -4,8 +4,3 @@ import { ActionContext } from 'vuex'
 export interface ActionIF {
```

Comment:
Is the "ActionIF" interface being used anywhere? This is the file @severinbeauvais was talking about deleting.

Reply:
Please also remove the reference to this file from the interfaces/index.ts file. It's causing a failure of the build and unit tests checks.

Before committing, you should do these to check for issues:
```
npm run lint
npm run test:unit
```

Reply:
removed reference!